drivers/arm/ccn/ccn.c

```diff
@@ -553,7 +553,13 @@ static unsigned int get_region_id_for_node(node_types_t node_type,
 		return REGION_ID_LIMIT;
 	}
 
-	region_id += node_pos_in_map;
+	/* For RN-I like nodes, region id depends on the node id
```

Comment:
Need more justification why this is the case. Could you clarify from the specification ? 

The assertion at line 539 seems to suggest that this node must be present in the bitmap. Is it the case that RN-I node_bitmap is (oly_rni_nodeid_list | oly_rnidvm_nodeid_list) ?

Also the comment should follow multi line comment format used at line line 526

Reply:
![image](https://user-images.githubusercontent.com/17290751/49778894-671b5400-fd2d-11e8-8602-d355841aa227.png)

The above snapshot is taken from ARM CCN-508 TRM.
You can see that, for RN-I, the region offset uses node id instead of node position.

Reply:
would be good to have this reference in the code comments above.

Reply:
Comments updated.